### PR TITLE
Consolidate ProgressRingControl class (currently duplicated)

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
The ProgressRingControl class was copied from SharedUx into CommonUxControls, but the SharedUx copy was never removed. This PR removes the SharedUx version and points all SharedUx.ProgressRingControl references to the CommonUxControls.ProgressRingControl.

#### PR checklist

- [x] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes? UI should be identical after the change
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights/Modes/LiveModeControl.xaml [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
                 <TextBlock Width="Auto" Height="Auto" VerticalAlignment="Top" HorizontalAlignment="Stretch" Margin="16" FontStyle="Italic" FontSize="{DynamicResource StandardTextSize}" Style="{StaticResource TbFocusable}">
                     <Run Text="{x:Static properties:Resources.LiveModeControl_RunAutomatedChecks}"/>
-                    <controls1:FabricIconControl GlyphName="TestBeaker" GlyphSize="Custom" FontSize="{DynamicResource StandardTextSize}" Margin="0,-2" Foreground="{DynamicResource ResourceKey=IconBrush}" FontStyle="Normal"/>
+                    <fabric:FabricIconControl GlyphName="TestBeaker" GlyphSize="Custom" FontSize="{DynamicResource StandardTextSize}" Margin="0,-2" Foreground="{DynamicResource ResourceKey=IconBrush}" FontStyle="Normal"/>
```
Comment (DaveTryon) [inline src/AccessibilityInsights/Modes/LiveModeControl.xaml]: >fabric [](start = 21, length = 6)

I changed `controls1` to `fabric` to be consistent with other places where this namespace was referenced

File: src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml [hunk 2]
```diff
@@ -7,9 +7,9 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:fabric="clr-namespace:AccessibilityInsights.CommonUxComponents.Controls;assembly=AccessibilityInsights.CommonUxComponents"
```
Comment (DaveTryon) [inline src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml]: >fabric [](start = 10, length = 6)

I changed `controls` to `fabric` to be consistent with other places where this namespace was referenced, and to reserve `controls` for `AccessibilityInsights.SharedUx.Controls`

File: src/AccessibilityInsights.SharedUx/Controls/ProgressRingControl.xaml.cs [hunk 3]
```diff
@@ -1,278 +0,0 @@
...
-        {
-            return (IsInternalScreenReaderActive() || IsExternalScreenReaderActive()) && AutomationPeer.ListenerExists(AutomationEvents.LiveRegionChanged);
-        }
-
-        /// <summary>
-        /// Check if we need to play scanning sound, play sound while meet both conditions below:
-        ///    1. "play sound while scanning" option in settings pannel is on
-        ///    2. At least one AT tool is running
-        /// </summary>
-        public static bool ShouldPlayScannerSound()
```
Comment (jalkire) [inline src/AccessibilityInsights.SharedUx/Controls/ProgressRingControl.xaml.cs]: `ShouldPlayScannerSound` in the CommonUxComponents version of `ProgressRingControl` does not access the user preference for `PlayScanningSound`. There is currently no way for CommonUxComponents to access user preferences, and I think it should stay that way. One option is to add a DependencyProperty to `ProgressRingControl` that allows you to pass in the config value from the hosting control.

**Comment by DaveTryon** [comment]
Closing this for now. @jalkire will probably pick it up in a separate PR

**Pull request closed without merging**